boards/arm/96b_carbon_nrf51/96b_carbon_nrf51-pinctrl.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default: uart0_default {
+		pins1 {
```

Comment:
should we use `group1..N` now? or is `pins1..N` just ok?

Reply:
I think `group` fits better.